config/connect-avro-distributed.properties

```diff
@@ -57,7 +57,26 @@ internal.value.converter=org.apache.kafka.connect.json.JsonConverter
 internal.key.converter.schemas.enable=false
 internal.value.converter.schemas.enable=false
 
-# Confuent Control Center Integration -- uncomment these lines to enable Kafka client interceptors
+# Confluent Control Center Integration -- uncomment these lines to enable Kafka client interceptors
 # that will report audit data that can be displayed and analyzed in Confluent Control Center
 # producer.interceptor.classes=io.confluent.monitoring.clients.interceptor.MonitoringProducerInterceptor
 # consumer.interceptor.classes=io.confluent.monitoring.clients.interceptor.MonitoringConsumerInterceptor
+
+# These are provided to inform the user about the presence of the REST host and port configs
```

Comment:
Not sure I understand, isn't this an addition to both? My confusion is that this comment seems like another meta-comment to help *us* on top of the normal user-oriented comment that follows on the next line.

When a comment says something about "inform the user" it feels like internal documentation that wouldn't make sense for a user-facing config file.